oat-sa/extension-tao-itemqti-pci
# Fix/tr 5243 add allow copy to mathentry

Related to https://oat-sa.atlassian.net/browse/TR-5243

- Add `data-allow-copy` attribute to indicate to disable commands that copy-paste is allowed.
https://github.com/oat-sa/tao-deliver-fe/blob/develop/packages/test-runner/src/runner/plugins/security/disableCommands/helpers.js#L38

Test:
- Create a test with MathEntry and run it is solar with enabled disableCommands, like:
```
{
    "https://purl.imsglobal.org/spec/lti/claim/roles": [
        "http://purl.imsglobal.org/vocab/lis/v2/membership#Learner"
    ],
    "https://purl.imsglobal.org/spec/lti/claim/custom": {
        "deliverySettings.plugins.disableCommands.enabled": true
    }
}
```
- Check migration script
